include/sdlmain.h

```diff
@@ -337,4 +337,18 @@ struct SDL_Block {
 
 extern SDL_Block sdl;
 
+inline uint32_t sdl_version_to_uint32(const SDL_version version)
```

Comment:
good candidate for constexpr?

`constexpr uint32_t sdl_version_to_uint32(const SDL_version version)`

Reply:
Yeah, why not 😄